build/common.sh

```diff
@@ -541,6 +538,11 @@ function kube::build::clean_images() {
 }
 
 function kube::build::ensure_data_container() {
+  # This is temporary, while last remnants of _workspace are obliterated
```

Comment:
What is this about?  In case there are cached old builds and the cached containers are tainted?